components/chainhook-cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "chainhook"
-version = "0.20.0"
+version = "1.0.0"
 edition = "2021"
```

Comment:
A naive-question. Is this intentional to be 2021?

Reply:
Yes, this is the rust edition, not the edition of this tool